app/controllers/users_controller.rb

```diff
@@ -1,73 +1,73 @@
+# frozen_string_literal: true
+
 class UsersController < ApplicationController
-  before_action :authenticate_user!, :permitted_params
+  before_action :authenticate_user!, except: :cities
+  before_action :permitted_params, except: :cities
 
   def show
-    unless current_user
-      redirect_to root_path && return
-    end
+    redirect_to root_path && return unless current_user
     @user = User.find_by_url_slug(permitted_params[:url_slug])
 
-    unless @user.present?
-      redirect_to root_path && return
-    end
+    redirect_to root_path && return unless @user.present?
 
-    review = Review.where(:author_id => current_user.id, :user_id => @user.id).first
+    review = Review.where(author_id: current_user.id, user_id: @user.id).first
 
-    review = ReviewDecorator.new(review, {
-        reviewee_first_name: @user.first_name,
-        reviewer: current_user
-    }).simple_decorate
+    review = ReviewDecorator.new(review,
+                                 reviewee_first_name: @user.first_name,
+                                 reviewer: current_user).simple_decorate
 
     @comments = get_ten_last_comments
 
-    @availabilities = Availability.where(:user => @user.id)
+    @availabilities = Availability.where(user: @user.id)
 
     normalize_for_current_user
 
     @data = {
```

Comment:
are these rubocop suggestions ?

Reply:
Yes.  Rubocop likes the more succinct syntax of `key:` instead of `:key =>`

In an attempt to start to clean up the Rails code, I ran Rubocop -a (auto fix) on each ruby file that I changed.   For errors/warnings that could not be automatically fixed, I did not make any changes.